File: easy/longestWord/longestWord.py
# Makes Python 2.x and 3.x play nice.
try: input = raw_input
except NameError: pass

def longestWord(sentence):
    words = sentence.split(" ")
    longest = ""
    max = 0
    for word in words:
        count = 0
        temp = ""
        for c in word:
            if (c >= 'a' and c <= 'z') or (c >= 'A' and c <= 'Z'):
                count += 1
                temp += c
        if (count > max):
            longest = temp
            max = count
    return longest

print(longestWord(input("Enter a String: ")))

File: easy/longestWord/longestWord.java
import java.util.Scanner;

public class longestWord {
	public static void main(String[]args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("Enter a String: ");
		System.out.println(longestWord(scan.nextLine()));
	}

	public static String longestWord(String sentence) {
		String longest = "";
		int max = 0;
		for (String word : sentence.split(" ")) {
			String temp = "";
			int count = 0;
			for (char c : word.toCharArray()) {
				if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z')) {
					count++;
					temp += c;
				}
			}
			if (count > max) {
				longest = temp;
				max = count;
			}
		}
		return longest;
	}
}